### ISSUE TITLE
Mojang Loading Screen Loops

### ISSUE THREAD

**Issue by Jqsmine** [jellysquid3/Sodium#725]
### Version

sodium-fabric-mc1.16.3-0.1.0.jar

### Reproduction Steps

1. add sodium mod to 'all of fabric 3' mods folder in curseforge
2. load the Minecraft launcher and click play
3. Mojang loading screen loads, makes ding sound and repeats. It never bring you to the main menu

### Crash Report File
<img width="1008" alt="Screen Shot 2021-07-01 at 4 50 18 pm" src="https://user-images.githubusercontent.com/86766604/124120875-99d9f300-dab7-11eb-99c2-3fc168981b6d.png">
<img width="982" alt="Screen Shot 2021-07-01 at 4 50 51 pm" src="https://user-images.githubusercontent.com/86766604/124120881-9ba3b680-dab7-11eb-8c44-cab826b6b9ae.png">

[latest.log](https://github.com/CaffeineMC/sodium-fabric/files/6747873/latest.log)

the game did not crash. it just doesn't load.
1st screenshot is the loading screen that keep recurring
2nd screenshot is the small glimpse of the main menu (it is unresponsive) that displays on screen for approx 1 second before starting the error again.

### Additional Information

When I take sodium file out of the game, it loads fine without error but when it is in the mods folder, the error occurs.

**Comment by LucilleTea** [comment]
Duplicate of #5 
To work around it you need to use Indium, which is another mod not by us. The AoF3 discord has resources for this.

**Comment by Jqsmine** [comment]
> Duplicate of #5
> To work around it you need to use Indium, which is another mod not by us. The AoF3 discord has resources for this.

do I keep sodium in the modpack aswell?
